### ISSUE TITLE
[Feature]: title

### ISSUE THREAD

**Issue by burrinhoMAC** [pot-app/pot-desktop#288]
### Please confirm the following before submitting a new issue.

- [X] I have carefully reviewed the official website's usage documentation.
- [X] I have searched through the historical issues but could not find an answer.

### Description

Congratulations on the excellent application.
Can you add this feature to paste translated text like Qtranslate?

### Application Scenario

Paste translated text.

### References

Qtranslate.

**Comment by Pylogmon** [comment]
Please provide a detailed description of what "Paste translated text" operation refers to. Currently, Pot supports automatically copying the translated text and also provides a one-click copy button. Is this the functionality you are referring to?

**Comment by burrinhoMAC** [comment]
No. Tha app Qtranslate have a button that replace the original text with tranlated text.

**Comment by burrinhoMAC** [comment]
https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTvJWo3jmIigYiu6L7mNBWlII3m0ZIMocuPgw&usqp=CAU![images (14).jpg](https://github.com/pot-app/pot-desktop/assets/110316011/ae235474-ec2f-4a8d-9635-29d0565d2288)

**Comment by Pylogmon** [comment]
Well, I understand what you mean, but I'm not sure how to implement such a feature. It can be quite challenging to achieve this functionality for cross-platform software.

**Comment by burrinhoMAC** [comment]
I believe that button is only "CTRL+C" and "CTRL+V"

> ![images (14).jpg](https://user-images.githubusercontent.com/110316011/252151673-bcc96378-0ecb-4011-997f-88406c119e0e.jpg)

**Comment by Pylogmon** [comment]
Once the translation is done, the focus of the interface has changed, I'm not sure if direct emulation of Ctrl+V will work on all platforms and in all scenarios.
